app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,18 @@
+module DeviseHelper
+  def devise_error_messages!
+    return '' if resource.errors.empty?
+    render 'devise/shared/errors_massages'.html_safe
```

Comment:
`#html_safe` is not needed here, as u making only partial path string as HTML safe, not a rendered HTML